process_config_test.go

```diff
@@ -45,54 +44,3 @@ Template
 		})
 	}
 }
-
-func TestNewConfigurationProcessor(t *testing.T) {
```

Comment:
Why is this being removed?

Reply:
Since we added other auth methods, Github token is not required anymore.